deps/node/lib/internal/lwnode/setup.js

```diff
@@ -95,6 +95,15 @@ function wrapLWNodeMethods(binding) {
         return binding.hasSystemInfo.apply(null, args);
       }
     },
+    binding: (message) => {
+      if (typeof message !== "string") {
+        throw new TypeError("The message argument must be a string");
+      }
+
+      if (binding.binding) {
+        return binding.binding(message);
```

Comment:
It would be better to rename this function name that starts with a verb. `lwnode.binding("something");` looks to load the native module, 'something'.

Reply:
I changed the name to what we discussed.